install/0000_80_machine-config-operator_02_images.configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   images.json: >
     {
+      "releaseVersion": "0.0.1-snapshot",
```

Comment:
Is there something that substitutes this value?  In `oc adm release new`?  I am not seeing it in `$ rg releaseVersion` in oc git master.

Reply:
`oc adm release new` is substituting it according to my manual testing, if you can double check that , it would be amazing

Reply:
also, grep for `0.0.1-snapshot` in openshift/origin - that's where the substitution is

Reply:
https://github.com/openshift/oc/blob/master/pkg/cli/admin/release/image_mapper.go#L317

Reply:
Also, https://github.com/openshift/origin/pull/23626